src/Tribe/Customizer.php

```diff
@@ -583,11 +583,13 @@ public function register( WP_Customize_Manager $customizer ) {
 			 * Allows people to Register and de-register the method to register more Fields
 			 *
 			 * @since 4.4
+			 * @since TBD Add Customizer instance as a parameter.
 			 *
 			 * @param array                $section
 			 * @param WP_Customize_Manager $manager
+			 * @param Tribe__Customizer    $customizer The current customizer instance.
 			 */
-			do_action( "tribe_customizer_register_{$id}_settings", $this->sections[ $id ], $this->manager );
+			do_action( "tribe_customizer_register_{$id}_settings", $this->sections[ $id ], $this->manager, $this );
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Line exceeds maximum limit of 100 characters; contains 115 characters

<sub>hash: 291e1cc445bc0420fc880f0e4599e8bf</sub>